### ISSUE TITLE
Search bar missing in certain firefox browser

### ISSUE THREAD

**Issue by htwangtw** [squidfunk/mkdocs-material#2339]
- [x] I've read the [contribution guidelines][1] and agree with them

__I've found a bug and checked that ...__

- [ ] ... the problem doesn't occur with the default MkDocs template
- [x] ... the problem is not in any of my customizations (CSS, JS, template)
- [x] ... the documentation does not mention anything about my problem
- [x] ... there are no open or closed issues that are related to my problem

## Description

Search bar doesn't show up with mkdocs-material 7.0.0 rendered site. 
This issue is redirected from https://github.com/bids-standard/bids-specification/issues/744

### Expected behavior

Firefox: 85.0.1
![image](https://user-images.githubusercontent.com/6961185/108987542-b393a580-7693-11eb-8254-1c49c50a7417.png)

### Actual behavior
This is what I see:
![image](https://user-images.githubusercontent.com/13743617/109028450-b99c7d00-76b9-11eb-9386-07e2395cece8.png)

Same behaviour with https://squidfunk.github.io/mkdocs-material/ 
![image](https://user-images.githubusercontent.com/13743617/109028265-89ed7500-76b9-11eb-9d66-2b13441b2ac0.png)

### Steps to reproduce the bug

N/A

### Package versions

See: https://github.com/bids-standard/bids-specification

### Project configuration

See: https://github.com/bids-standard/bids-specification/blob/master/mkdocs.yml

### System information

* OS: Debian 10
* Browser: Firefox 78.7.0esr (latest for Debian)

  [1]: https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md

**Comment by squidfunk** [comment]
Could you please check the developer console? I suspect that Firefox 78 is too old and is missing an ES6/7 feature we're using.

**Comment by htwangtw** [comment]
The version of the website built with 6.2.8 is fine (https://bids-specification.readthedocs.io/en/v1.4.1/)
Can you explain a bit more what should I check in the developer console?

**Comment by squidfunk** [comment]
Please check whether there are any errors listed below the "Console" tab in the [Firefox Developer Console](https://developer.mozilla.org/en-US/docs/Tools/Browser_Console). If unsure, consult another developer on your team to validate.

**Comment by htwangtw** [comment]
There are two messages when I check the BIDS website:
Error:
```
Uncaught SyntaxError: expected expression, got '='bundle.3981fc68.min.js:3:48064
```
This message is not listed as an error but I thought it might be relevant:
```
Server side search is disabled. readthedocs-doc-embed.js:1:28238
```

Just cross-checking with the official page of Material, we got the same error as above, and an extra one:
```
Uncaught ReferenceError: document$ is not defined
    <anonymous> bundle.ts:40
    <anonymous> bundle.ts:49
bundle.ts:40
    <anonymous> bundle.ts:40
    <anonymous> bundle.ts:49

```

**Comment by squidfunk** [comment]
Which website are you referring to? When I look at both links you posted in https://github.com/bids-standard/bids-specification/issues/744, there are entirely different chunkhashes:

- `bundle.926459b3.min.js:`
- `bundle.4e3fea51.min.js`

Please provide complete information, i.e. also attach the file for which this happens, or we have no possibility to debug this. If unsure, please consult a developer colleague.

**Comment by divansantana** [comment]
have same issue with 7.0.2

dev tools in my qutebrowser says:

```
Uncaught SyntaxError: Unexcted token '='
bundle.3981fc68.min.js:3
```

browser version:
```
qutebrowser v1.14.1
Git commit: 
Backend: QtWebEngine (Chromium 83.0.4103.122)
Qt: 5.15.2
```

Tried `reload -f` but didn't help.

**Comment by divansantana** [comment]
I think it might be a browser version thing. Some users report after upgrading browser it works.

**Comment by lyz-code** [comment]
I've got the same error [here](https://lyz-code.github.io/blue-book). 

The developer console shows:

`Uncaught SyntaxError: expected expression, got '='    bundle.4e3fea51.min.js:3:48067`

I've reproduced it with the following browsers:

* Firefox 78.5.0esr
* Chromium 83.0.4103.116
* Firefox Klar 8.4.0

So I'll roll out that this is a browser issue.

**Comment by squidfunk** [comment]
The transpile target was `es2020`. 500d5e6f set it at `es2015`, which should fix the issues. Please try the latest `master`, so we can issue a bugfix release.

**Comment by squidfunk** [comment]
... you could also just check https://squidfunk.github.io/mkdocs-material/ – if that works, we can issue the release.

**Comment by htwangtw** [comment]
The search bar is showing with Firefox 78.5.0esr.

**Comment by squidfunk** [comment]
Perfect, thanks!

**Comment by squidfunk** [comment]
Released as part of 7.0.3.

**Comment by effigies** [comment]
Thanks for the quick turnaround, @squidfunk!
